qiskit_quantinuum/quantinuumbackend.py

```diff
@@ -24,36 +24,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Module for interfacing with a Honeywell Backend."""
+"""Module for interfacing with a Quantinuum Backend."""
 
 import logging
 import warnings
 
 from qiskit.circuit import QuantumCircuit
-from qiskit.exceptions import QiskitError
 from qiskit.providers import BackendV1
 from qiskit.providers.models import BackendStatus
 from qiskit.providers import Options
 from qiskit.utils import deprecate_arguments
 from qiskit import qobj as qobj_mod
 from qiskit import pulse
 
-from .honeywelljob import HoneywellJob
+from qiskit_quantinuum.exceptions import QiskitError
+from .quantinuumjob import QuantinuumJob
 
 logger = logging.getLogger(__name__)
 
 
-class HoneywellBackend(BackendV1):
-    """Backend class interfacing with a Honeywell backend."""
+class QuantinuumBackend(BackendV1):
```

Comment:
This is a note for the future, but we should look at moving this to `BackendV2` in a follow up PR.